web/.env.example

```
@@ -2,6 +2,14 @@ RESEND_API_KEY=
 CMUX_FEEDBACK_FROM_EMAIL=
 CMUX_FEEDBACK_RATE_LIMIT_ID=
 
+# cmux Founder's Edition welcome email. /api/stripe/founders-welcome verifies the
+# Stripe webhook signature with this endpoint secret, then sends the welcome email
+# via Resend (To customer, CC the founders). Leave the secret empty to disable the
+# route. CMUX_FOUNDERS_FROM_EMAIL overrides the sender (defaults to
+# austin@manaflow.ai) and must be on a Resend-verified domain.
+STRIPE_FOUNDERS_WEBHOOK_SECRET=
+CMUX_FOUNDERS_FROM_EMAIL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CMUX_FOUNDERS_FROM_EMAIL key should go before the STRIPE_FOUNDERS_WEBHOOK_SECRET key

(UnorderedKey)